greenplumpython/func.py

```diff
@@ -97,6 +130,52 @@ def make_function_call(
             ),
             has_results=False,
         )
-        return FunctionCall(qualified_func_name, args, as_name=as_name, db=db)
+        return FunctionCall(qualified_func_name, args=args, as_name=as_name, db=db)
+
+    return make_function_call
+
+
+def create_aggregate(
+    trans_func: Callable,
+    name: Optional[str] = None,
+    schema: Optional[str] = None,
+    temp: bool = True,
+    replace_if_exists: bool = False,
+    language_handler: str = "plpython3u",
```

Comment:
None of those arguments has test cases. Add FIXME if you plan to add them later.

Reply:
FIXME added